### PR TITLE
Enable e2e's for News Sport Newsround AMP

### PR DESCRIPTION
**Pull request by paruchurisilpa**
Resolves #https://github.com/bbc/simorgh/issues/8960

**Overall change:**
- Added Fixture data for CSP and STY for News, Sport and Newsround
- Enable News, Sport, Newsround e2e tests for AMP pages but not run on Canonical pages

---

- [ x ] I have assigned myself to this PR and the corresponding issues
- [ x ] I have assigned this PR to the Simorgh project

**Testing:**

- [ x ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)

### CODE REVIEW

**Review by LilyL0u — APPROVED** [review]

File: cypress/support/helpers/runTestsForPage.js [hunk 1]
```diff
@@ -55,29 +56,33 @@ const runTestsForPage = ({
             variant: config[service].variant,
           };
 
-          // Enables overriding of the smoke test values in the config/settings.js file
-          testsThatAlwaysRunForAllPages(testArgs);
-          testsThatAlwaysRunForAllCanonicalPages(testArgs);
-          // Page specific tests
-          testsThatAlwaysRunForCanonicalOnly(testArgs);
-          testsThatAlwaysRun(testArgs);
+          const servicesToNotRunCanonical = ['news', 'sport', 'newsround'];
```
Comment (andrewscfc) [inline cypress/support/helpers/runTestsForPage.js]:
```suggestion
          const ampOnlyServices = ['news', 'sport', 'newsround'];
```

File: cypress/support/helpers/runTestsForPage.js [hunk 2]
```diff
@@ -55,29 +56,33 @@ const runTestsForPage = ({
...
 
-          // This runs most tests but only on Service:PageType combinations with smoke enabled
-          if (shouldSmokeTest(pageType, service)) {
-            testsThatFollowSmokeTestConfigforAllPages(testArgs);
-            testsThatFollowSmokeTestConfigForAllCanonicalPages(testArgs);
+          if (!servicesToNotRunCanonical.includes(service)) {
```
Comment (andrewscfc) [inline cypress/support/helpers/runTestsForPage.js]:
```suggestion
          if (!ampOnlyServices.includes(service)) {
```

**Review by andrewscfc — APPROVED** [review]
One naming suggestion otherwise LGTM, nice work!

**Comment by paruchurisilpa** [comment]
> One naming suggestion otherwise LGTM, nice work!

Name change done..

**Pull request merged**